src/ParserUtils.idr

```diff
@@ -0,0 +1,17 @@
+module ParserUtils
```

Comment:
This looks general enough to become a PR for TParsec: `except` could go into `Combinators` and `notChar` into `Combinators.Chars`

Reply:
I've opened a pull request (https://github.com/gallais/idris-tparsec/pull/20). I suggest we merge this PR and we change the dependencies ones TParsec is updated. What do you think?

Reply:
Yeah, sounds good